### PR TITLE
fix: add new link component

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
fixes: https://github.com/SAP/fundamental-ngx/issues/1843
#### Please provide a brief summary of this pull request.
There is completely new component named link, with properties following `fundamental-styles`
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by InnaAtanasova** [comment]
LGTM 
*I will approve the PR after we discuss the point I raised in Slack

**Comment by InnaAtanasova** [comment]
@Vanessa-Cusmich it's a new component. Do you mind reviewing the English? Thank you

File: libs/core/src/lib/link/link.component.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import { applyCssClass, CssClassBuilder } from '../utils/public_api';
+
+@Component({
+    // tslint:disable-next-line:component-selector
+    selector: '[fdLink], [fd-link]',
+    template: `
+        <ng-content></ng-content>
+    `,
+    styleUrls: ['./link.component.scss'],
+    encapsulation: ViewEncapsulation.None,
```
Comment (salarenko) [inline libs/core/src/lib/link/link.component.ts]: ChangeDetectionStrategy.OnPush

Comment (stefanoScalzo) [reply libs/core/src/lib/link/link.component.ts]: Agreed

**Pull request merged**